.drone.yml

```diff
@@ -220,6 +305,7 @@ steps:
     platforms:
       - linux/amd64
       - linux/arm64
+      - linux/s390x
```

Comment:
`- linux/s390x` is missed in the above manifest step for publishing the head image.

Reply:
Missing line added in in https://github.com/longhorn/longhorn-engine/pull/693/commits/72ebdedb292162dee2dc794ce405154ecc00bf0d